src/GuildsApp.tsx

```diff
@@ -74,6 +78,25 @@ const GuildsApp = () => {
                       </EtherSWRConfig>
                     </GuildsContextProvider>
                   </Route>
+                  <Route exact path="/:chain_name/:guild_id/ProposalType">
+                    <GuildsContextProvider>

```

Comment:
I know this is not specific from your pr, but shouldn't we add one single GuildsContextProvider (maybe after transactionProvider or something like that) instead of adding a new contextProvider per route?

Reply:
Yeah I thought about as well but that would be outside the scope of this PR, so I didn't want to change anything. I'll make an issue about it.